e2e/nomostest/ntopts/gke.go

```diff
@@ -27,8 +29,18 @@ import (
 func GKECluster(t testing.NTB) Opt {
```

Comment:
Suggest extracting a GKECluster struct with CRUD methods to a sub-package.

Reply:
I moved GKE/Kind out to a sub-package and have some ideas to define a common interface that can be implemented by both GKE and Kind clusters. This is looking like a bigger change though so I'd like to implement that as a followup PR